packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener-rule.ts

```diff
@@ -231,46 +253,66 @@ export class ApplicationListenerRule extends cdk.Construct {
     this.conditions[field] = values;
   }
 
+  /**
+   * Configure the action to perform for this rule
+   */
+  public configureAction(action: ListenerAction) {
+    // I'd like to throw here but there might be existing programs that happen
+    // to work even though they followed an illegal call pattern. Just add a warning.
+    if (this.action) {
+      this.node.addWarning('An Action already existed on this ListenerRule and was replaced. Configure exactly one default Action.');
+    }
```

Comment:
Not sure I follow why you're not throwing here.

Reply:
Added more explanation.